relay-server/src/metrics_extraction/transactions/mod.rs

```diff
@@ -873,7 +890,7 @@ mod tests {
             "release": "1.2.3",
             "dist": "foo ",
             "environment": "fake_environment",
-            "transaction": "mytransaction",
+            "transaction": "GET /api/:version/users/",
```

Comment:
Note that these first metrics are transaction metrics, so `transaction.method` is expected to not exist in those.